### PR TITLE
Create abwaocode.json

### PR DESCRIPTION
**Pull request by abwaocode**
null

### CODE REVIEW

File: domains/abwaocode.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "abwaocode",
+        "email": "abwaocode.address"
```
Comment (cupglassDEV) [inline domains/abwaocode.json:4]: This isnt valid, it should be your email, maybe like this
```json
        "email": "abwaocode@gmail.com"
```

**Review by ghost — COMMENTED** [review]
please check the remaining issue

File: domains/abwaocode.json (hunk 1 above)
Comment (Stef-00012) [inline domains/abwaocode.json:4]: email should be your email address

File: domains/abwaocode.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    },
+    "record": {
+        "CNAME": "abwaocode.github.io"
```
Comment (Stef-00012) [inline domains/abwaocode.json:7]: I cannot reach your website, 404 and i see no other repo on your profile with github pages enabled

you gave as preview url https://abwaocode.serv00.net/ and then used github pages which is a different host, in no way github pages cname would work on that other host
anyway for https://abwaocode.serv00.net/ I wouldn't consider that complete

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**